charts/fulcio/templates/fulcio-configmap.yaml

```diff
@@ -9,16 +9,21 @@ data:
   config.json: |-
     {
       "OIDCIssuers": {
```

Comment:
How about we enable the entire `config.json` to be defined via a yaml value. if not defined, we can use the boilerplate default. otherwise, we can substitute the user provided content that would be converted to JSON using the `toJson` function.

Is this what you had in mind with the `config.contents` property?